horreum-backend/src/test/java/io/hyperfoil/tools/horreum/svc/TestServiceTest.java

```diff
@@ -613,6 +614,7 @@ private TestService.TestQueryResult listTests(String token, String roles, Intege
     }
 
     @org.junit.jupiter.api.Test
+    @Ignore
```

Comment:
If you want to ignore a test I think you should use `@Disabled` instead of `@Ignore`. But I was wondering why this started to fail, is it because of some internal changes due to the removal of the `executeBlocking` ?

Reply:
I need to verify, but the recalculation seemed to take longer and the sleep I had previously was not enough to match the expected labelValues. I added a poller now which hopefully should work better.